tests/commands/test-kob-status.sh

```diff
@@ -1,106 +1,173 @@
 #!/bin/bash
 
-status="true"
-
+path_to_kob_envs="${KOBMAN_DIR}/envs"
+environment=$1
+version=$2
+if [[ -z $environment || -z $version ]]; then
+    echo "Usage: ./test-kob-status.sh <env_name> <version>"
+    exit 1
+fi
 function __test_kob_init
 {
-    echo "Checking for kob..."
     if [[ -d $KOBMAN_DIR ]]; then
         source $KOBMAN_DIR/src/kobman-utils.sh
         __kobman_echo_no_colour "kob found"
         source $KOBMAN_DIR/bin/kobman-init.sh
     else
-        status="false"
         echo "kob not found"
-        echo "Please install KOBman first and try again"
-        echo "Exiting!!!"
+        echo "Please install KOBman and try again"
+        echo "Exiting!!1"
         exit 1
-        
-    fi
-    
-    __kobman_echo_no_colour "Checking for installed environments...."
-    env_folder=($(find $KOBMAN_DIR/envs/ -name "kob_env_*" -print))
-    if [[ -z $env_folder ]]; then
-        __kobman_echo_no_colour "No environments installed"
-        __kobman_echo_no_colour "Please install an enivornment first and try again"
-        __kobman_echo_no_colour "Exiting!!!"
-        status="false"
-        return 1
-    else
-        __kobman_echo_no_colour "Environments found"
-        __kobman_echo_no_colour "Proceeding with the test..."
     fi
 
+    export INTERACTIVE_MODE="false"
```

Comment:
Please rename INTERACTIVE_MODE to KOBMAN_INTERACTIVE_USER_MODE, all KOBman export variables will have KOBMAN_ prefix.